tests/console/validate_unencrypted_partition.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validation module to check that partition is inactive.
```

Comment:
I would emphasize that we check that luks is not activated. Partition itself should be visible and available.

Reply:
changed it to "Validate that hard disk encryption(LUKS) is not activated on the configured partitioning". hopefully is better